cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -504,6 +518,8 @@ TEST(Substrait, SupportedLiterals) {
     ASSERT_OK_AND_ASSIGN(auto json, internal::SubstraitToJSON("Type", *buf));
     ExpectEq("{\"null\": " + json + "}", MakeNullScalar(type));
   }
+
+  ConversionOptions conversion_options;
```

Comment:
This seems unused, did you mean to put it in `ExpectEq` with 
`conversion_options.strictness = ConversionStrictness::BEST_EFFORT`?

Reply:
Nope, this is just leftover from an earlier iteration where I was testing nullability outside of `ExpectEq`.  Thanks for catching.  I've removed it.